<launch>
    <rosparam file="$(find industrial_calibration_ros)/config/collection_config.yaml" />
    <node name="data_collection_node" pkg="industrial_calibration_ros" type="data_collection_node.py" output="screen"/>
</launch>